docs/content/docs/guide/resolver.md

```diff
@@ -191,7 +191,32 @@ or from a relative path inside the repository:
 pipelinesascode.tekton.dev/pipeline: "./tasks/pipeline.yaml
 ```
 
-Fetching `Pipelines` from the [Tekton Hub](https://hub.tekton.dev) is not currently supported.
+### [Tekton Hub](https://hub.tekton.dev) Support for Pipelines
```

Comment:
maybe mentions as well here that only one pipeline is supported

Reply:
It was there already before this section, I have rephrased that. Thanks